cit3.0-web/src/components/OpportunityListItem/OpportunityListItem.js

```diff
@@ -70,7 +70,10 @@ const OpportunityListItem = ({ opportunity, publicView, handleModalOpen }) => {
 
   return (
     <div key={opportunity.id} className="opportunity-table-row w-100">
-      <Row>
+      <Row
+        className={publicView ? "nested-link" : ""}
+        onClick={() => history.push(opportunity.link)}
```

Comment:
This allows you to click on the opportunity to open it in either the public or private lists, but the private one doesn't have the hover pointer or link highlighting CSS. Is that intentional?

Reply:
Nope, not intentional, let me change that.